doc/api/build.md

```diff
@@ -70,3 +70,40 @@ GET /api/application/{application_id}/build/latest
   "download_link": "http://app-build.dev/app_dev.php/fr/application/1/build/1/download"
 }
 ```
+
+## Create a new build (WIP)
+
+### Request
+
+```
+PUT /api/application/{application_id}/build
+```
+```json
+{
+  "version": "1.5.2",
+  "comment": "This is a comment for this build.\nWith a line break.\nAnd another one.\n"
+}
+```
+
+### Response headers
+
+```
+Location: http://domain.com/application/3/build/1/file
+```
+
+## Upload a file for a build (WIP)
+
+Note: This route is given by `Create a new build` and can be put on an other domain (like amazon S3).
```

Comment:
j'suis pas sûr que `and can be put on an other domain (like amazon S3)` soit bien utile, car cette doc est surtout à destination des utilisateurs de l'API, et ce n'est pas eux qui vont choisir où pointera cette route, eux ils doivent se contenter d'utiliser ce qu'on leur donne